packages/libsql-serverless/README.md

```diff
@@ -0,0 +1,23 @@
+# libSQL serverless driver for TypeScript and JavaScript
+
+## Getting Started
+
+```typescript
+import { Database } from "@libsql/serverless"
+
+const db = new Database();
+await db.execute("CREATE TABLE users (email TEXT)");
+await db.execute("INSERT INTO users (email) VALUES ('alice@example.com')");
+await db.execute("INSERT INTO users (email) VALUES ('bob@example.com')");
+```
+
+## Features
+
+* SQLite JavaScript API
+* SQLite-backed local-only backend
```

Comment:
SQLite or LibSQL?

Reply:
SQLite right now. We want to switch to libSQL, of course, but there's no npm package for that.